dashboard/test/ui/features/spritelab/dancelab.feature

```diff
@@ -9,3 +9,13 @@ Feature: Dance Lab
     Then element "#runButton" is visible
     And element "#resetButton" is hidden
     And element "#song_selector" is visible
+
+  # This test requires cloudfront.
+  @no_circle
+  Scenario: Restricted audio content is protected
+    When I am on "http://studio.code.org/restricted/placeholder.txt"
+    Then page text does not contain "placeholder for testing"
+
+    When I am on "http://studio.code.org/dashboardapi/sign_cookies"
```

Comment:
Does `URLConvert` switch the domain name to the domain of the environment that the test is running on?

Reply:
Yes, here: https://github.com/code-dot-org/code-dot-org/blob/13dea43cd5bf89bb4c53cc02fe2e442e5332763a/dashboard/test/ui/features/step_definitions/steps.rb#L38-L46